README.md

```diff
@@ -31,11 +31,13 @@ certificate used by MQTT broker to connect nRF Cloud under your account. So, you
 need to prepare nRF Cloud API key.
 
 ```bash
-./cli.sh configure thirdParty/nrfcloud/apiKey <API key>
-./cli.sh initialize-nrfcloud-account
-./cli.sh create-health-check-device
+./cli.sh configure thirdParty/<account>/apiKey <API key>
+./cli.sh initialize-nrfcloud-account <account>
+./cli.sh create-health-check-device <account>
 ```
 
+**Note** Currently the supported accounts are `exeger` and `nordic`
```

Comment:
Should not be hard-coded. Adding or removing account should not involve source code changes.